apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibility.jsx

```diff
@@ -147,6 +159,17 @@ export default class AmazonFutureEngineerEligibility extends React.Component {
   };
 
   submitToAFE = () => {
+    firehoseClient.putRecord({
+      study: 'amazon-future-engineer-eligibility',
+      event: 'submit_to_afe',
+      data_json: JSON.stringify({
+        accountEmail: this.props.accountEmail,
+        accountSchoolId: this.props.schoolId,
+        formEmail: this.state.formData.email,
+        formSchoolId: this.state.formData.schoolId
+      })
```

Comment:
I wonder if instead of formEmail and formSchoolId, we ought to simply send all of `this.state.formData` as we do in the submit request below, so we have the most possible info if we need to diagnose issues with this step.

Reply:
Sounds reasonable -- I'm working through some details with Bryan in the spec on what we're trying to make sure we can measure, so I expect there will be some subsequent tweaks to what/where we log. I can change this as part of that.